ingestion-sink/src/test/java/com/mozilla/telemetry/ingestion/sink/util/PubsubTopics.java

```diff
@@ -67,15 +68,18 @@ public String getTopic(int index) {
   /**
    * Publish a {@link PubsubMessage} to the topic indicated by {@code index}.
    */
-  public String publish(int index, PubsubMessage message) {
+  public List<String> publish(int index, PubsubMessage... messages) {
```

Comment:
the return value was only used in one place, so i figured it would be fine to update this method in place to allow the sdk to batch messages for publishing.